src/iexamine.cpp

```diff
@@ -234,6 +234,24 @@ void iexamine::none( player &/*p*/, const tripoint &examp )
     add_msg( _( "That is a %s." ), get_map().name( examp ) );
 }
 
+bool iexamine::always_false( const tripoint &/*examp*/ )
+{
+    return false;
+}
+
+
+bool iexamine::always_true( const tripoint &/*examp*/ )
+{
+    return true;
+}
```

Comment:
I am not great at C++, but wouldn't these belong in a header file?

Reply:
They have forward declarations in a header and implementation in a source file.

They could be moved into a header (and the compile would be more able to inline them), but they don't need to be.

Reply:
Given that it compiled, I assumed that they didn't absolutely need to be - just possibly preferably would be.